SQLite Tests/TestHelper.swift

```diff
@@ -71,3 +72,16 @@ class SQLiteTestCase: XCTestCase {
     }
 
 }
+
+
+public class UniqueIdentifier : NSUUID, Value {
```

Comment:
Only because I already had the `UniqueIdentifier` class definition handy and it tests that there are no problems using a completely custom column datatype as the primary key.

I did a test that uses the email field which is a `String` to cover that case.